app/pages/sections/supplier/supplier/ui.test.js

```diff
@@ -41,16 +41,37 @@ const supplierData = {
   },
 };
 
-const mocks = () => {
-  nock(solutionsApiUrl)
-    .get('/api/v1/suppliers/supplier-1')
-    .reply(200, supplierData);
+const supplierDataFromDapi = {
```

Comment:
`supplierDataFromDapi` should this be Bapi?

Reply:
indeed it should - too many -apis!